shared/src/main/kotlin/gov/cdc/prime/reportstream/shared/SubmissionQueueMessage.kt

```diff
@@ -0,0 +1,5 @@
+package gov.cdc.prime.reportstream.shared
+
+import java.util.UUID
+
+data class SubmissionQueueMessage(val reportId: UUID, val blobUrl: String, val headers: Map<String, String>)
```

Comment:
You're probably in a better place to evaluate this when integrating it into the convert step, but should we special case `clientId` since we absolutely need that to start the next step?